docs/notebooks/qubit_mapping_guide.ipynb

```diff
@@ -0,0 +1,796 @@
+      "text/plain": [
+       "<Figure size 1082.05x535.111 with 1 Axes>"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "num_qubits = 5\n",
+    "\n",
+    "ghz_small = QuantumCircuit(num_qubits)\n",
```

Comment:
Circuit names can be more cohesive.

Some suggestions:
 
`ghz_5q`
`ghz_5q_routed`
`ghz_5q_routed_translated`
`ghz_5q_routed_translated_df`
`ghz_5q_mapped` or `ghz_5q_mapomatic`